text/0000-more-env-info.md

```diff
@@ -0,0 +1,58 @@
+- Feature Name: more-env-info
+- Start Date: 2017-08-09
+- RFC PR:
+- Rust Issue:
+
+# Summary
+[summary]: #summary
+
+Introduce more potentially useful environment-related functions to `std::env`.
+
+# Motivation
+[motivation]: #motivation
+
+Programs sometimes need to know about their working environment to do their job properly. For example, linebreak convention differs for Windows and *NIX. Such discrepancy can lead to problems easily, especially when a program needs to communicate with aged third-party libraries.
```

Comment:
Please escape the `*` here ("``\*NIX``"), the rest of the document is badly highlighted on GitHub because of this.